taqueria-plugin-tezos-client/index.ts

```diff
@@ -45,6 +45,11 @@ Plugin.create(i18n => ({
                     description: "The name of the sandbox to use",
                     required: false
                 }),
+                Option.create({
```

Comment:
We should update the documentation to add this functionality. What would the command look like using this?
`taq simulate file.tz `10` --entrypoint '%increment'`

Reply:
`taq simulate sample.tz '2' --storage '3' --entrypoint increment`